sql/database.sql

```diff
@@ -7899,6 +7899,7 @@ INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES (
 
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('lists', 'page_validation', 'Page Validation', 298);
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'add_edit_issue#theform', '/interface/patient_file/summary/add_edit_issue.php', 10, '{form_title:{presence: true}}', 0);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'encounters#new_encounter', '/interface/forms/newpatient/new.php', 20, '{pc_catid:{exclusion: ["_blank"]}}', 0);
```

Comment:
No. Because there are many forms that only have a name but no id, we chose to use the name of the form in the method implementation and not the id. Otherwise we would have to go around and add id to all forms. Is this ok?